models/unit/unit.go

```diff
@@ -17,14 +18,15 @@ type Type int
 
 // Enumerate all the unit types
 const (
-	TypeCode            Type = iota + 1 // 1 code
-	TypeIssues                          // 2 issues
-	TypePullRequests                    // 3 PRs
-	TypeReleases                        // 4 Releases
-	TypeWiki                            // 5 Wiki
-	TypeExternalWiki                    // 6 ExternalWiki
-	TypeExternalTracker                 // 7 ExternalTracker
-	TypeProjects                        // 8 Kanban board
+	TypeInvalid         Type = iota // 0 invalid
```

Comment:
So it's possible, because of errors or typos by the user that this Unit is used. Is their any way to show that to the user(e.g. in the UI that a invalid unit was passed along?)

Reply:
The definition is only for convenience to check permissions. Since no unit has a value of 0, it has no chance to be displayed in User Interface.